Fix misleading CSRF_TRUSTED_ORIGINS variable description (#3869)
mozilla/pontoon: docs/admin/deployment.rst

``CSRF_TRUSTED_ORIGINS``

>    Optional. A list of trusted origins for unsafe requests. It should contain the domains

Reviewer: Splitting up this suggestion because GitHub can't deal with it at once.

We should match the other list descriptions.
```suggestion
   Optional. A comma-separated list of trusted origins for unsafe requests.
```
